src/pages/validator/amplifier/gmp-tutorial.mdx

```diff
@@ -0,0 +1,95 @@
+# General Message Passing with Amplifier
+
+In this tutorial, you will learn how to execute a GMP call from the Ethereum Sepolia to the Avalanche Fuji testnets.
+
+## Prerequisites
+
+You will need:
+
+- An [`axelard` node](/validator/amplifier/integrator-onboarding) that is [connected to the devnet](/validator/amplifier/verifier-onboarding)
+- A [Solidity contract](https://github.com/cjcobb23/axelar-examples/blob/main/examples/evm/call-contract/CallContract.sol) deployed to the Ethereum Sepolia and Avalanche Fuji testnets
+
+## Helpful references
+
+- [Amplifier devnet contract addresses](https://github.com/axelarnetwork/axelar-contract-deployments/pull/187/files)
```

Comment:
Currently links to a PR, should have direct link to GitHub JSON file once that PR is submitted.